cue.mod/pkg/github.com/prometheus/prometheus/schema/alert.cue

```diff
@@ -5,8 +5,17 @@ import "github.com/grafana/grafana/cue/scuemata"
 Alert: scuemata.#Family & {
     lineages: [
         [
-            {
-                placeholder: string
+            { // 0.0
+                // TODO docs
+                alert: string
+                // TODO docs
+                expr: string
+                // TODO docs
+                "for": string
```

Comment:
This always has to be in quotes everywhere. Maybe with Prometheus v3 we should push for changing `for` to something else...

Reply:
A great argument on its own for scuemata - if we were to, say, change the key to `over`, it can encapsulate the logic for migrating all existing artifacts from `"for"` to `over`, on the fly (i.e. when Prom reads them from disk).